docs/Development/HowTo_ColorText.md

```diff
@@ -1,16 +0,0 @@
-# How To: Color Text in WheelMUD
-
-## Tagging
-We are using a very similar tagging scheme as MudOS.
-ASP.NET programmers will feel right at home.
-The ANSI codes are stored in the ANSI table in the WheelMUD.net.db SQLite database.
```

Comment:
We actually shouldn't delete this whole file - Other place(s) still point to this file, which would become broken link.  Maybe just this one line about the SQLite storage should get deleted or replaced?  (The "table" is in code only - we could say to have a look at `CodeSequenceMap` in `AnsiHandler.cs` for supported options instead?)

Basically the set of codes the image shows like `<%blue%>` _are_ still a way for us to colorize output.  This document isn't the best but I hope it can provide enough value to exist.  :)

Reply:
Ok I'll update that file.